File: Examples/CSharp/Articles/SetPresetWordArtStyle.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using Aspose.Cells.Drawing;

namespace Aspose.Cells.Examples.CSharp.Articles
{
    public class SetPresetWordArtStyle
    {
        public static void Run()
        {
            //Output directory
            string outputDir = RunExamples.Get_OutputDirectory();

            // ExStart:1
            // Create workbook object
            Workbook workbook = new Workbook();

            // Access first worksheet
            Worksheet worksheet = workbook.Worksheets[0];

            // Create a textbox with some text
            TextBox textbox = worksheet.Shapes.AddTextBox(0, 0, 0, 0, 100, 700);
            textbox.Text = "Aspose File Format APIs";
            textbox.Font.Size = 44;

            // Sets preset WordArt style to the text of the shape.
            FontSetting fntSetting = textbox.GetCharacters()[0] as FontSetting;
            fntSetting.SetWordArtStyle(PresetWordArtStyle.WordArtStyle3);

            // Save the workbook in xlsx format
            workbook.Save(outputDir + "outputSetPresetWordArtStyle.xlsx");
            // ExEnd:1

            Console.WriteLine("SetPresetWordArtStyle executed successfully.");
        }
    }
}

File: Examples/VisualBasic/Articles/SetPresetWordArtStyle.vb
Imports Aspose.Cells
Imports Aspose.Cells.Drawing

Namespace Articles
    Public Class SetPresetWordArtStyle
        Public Shared Sub Run()
            ' ExStart:1
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType)

            ' Create workbook object
            Dim wb As New Workbook()

            ' Access first worksheet
            Dim ws As Worksheet = wb.Worksheets(0)

            ' Create a textbox with some text
            Dim tb As TextBox = ws.Shapes.AddTextBox(0, 0, 0, 0, 100, 700)
            tb.Text = "Aspose File Format APIs"
            tb.Font.Size = 44

            ' Sets preset WordArt style to the text of the shape.
            Dim fntSetting As FontSetting = TryCast(tb.GetCharacters()(0), FontSetting)
            fntSetting.SetWordArtStyle(PresetWordArtStyle.WordArtStyle3)

            ' Save the workbook in xlsx format
            wb.Save(dataDir & Convert.ToString("output_out.xlsx"))
            ' ExEnd:1
        End Sub
    End Class
End Namespace